django/preview/urls.py

```diff
@@ -0,0 +1,12 @@
+from django.urls import include, path
+
+from preview.api.router import router
+from preview.api import views
+
+urlpatterns = [
+    path("", include(router.urls)),
+    path("preview/", views.PreviewEndpoint.as_view(), name="preview"),
+    path(
+        "delete-resources/", views.ResourceEndpoint.as_view(), name="delete-resources"
+    ),
```

Comment:
I don't think this route should be in the `preview` app

Reply:
Agreed. Maybe the app `preview` is not named well